_posts/2018-01-14-graphql-union-vs-interface.md

```diff
@@ -0,0 +1,320 @@
+---
+layout: post
+title: GraphQL: Union vs. Interface
+date: 2019-01-14
+author: ashkan
+categories: [graphql ruby]
+---
+
+<!--
+
+At Artsy we’ve been moving towards GraphQL for all of our new services. Acknowledging GraphLQ is a relatively new technology, we faced some challenging questions as we were developing one our most recent services.
+
+Naively as my first attempt to define GraphQL types and schemas, I naturally tried to map our database models to GraphQL types. While this may work for lot of cases, we may not be utilizing some of the useful features that comes with GraphQL that can make the consuming our data lot easier.
+
+
+-->
```

Comment:
These top two paragraphs are commented out and will not show - is that on purpose?

Reply:
no, i think i missed the `<!-- OUTLINE -->` thing